gh_pages/_source/session3/Coordinate-Transforms-using-TF.md

```diff
@@ -29,8 +29,8 @@ Specifically, edit the service callback inside the vision_node to transform the
  1. Add `tf2_ros::Buffer` and `tf2_ros::TransformListener` objects to the vision node (as class members variables). 
 
     ``` c++
-    #include <tf2/buffer.h>
-    #include <tf2/transform_listener.h>
+    #include <tf2_ros/buffer.h>
+    #include <tf2_ros/transform_listener.h>
```

Comment:
I think we also need the `tf2_geometry_msgs` header as well per #331 and the exercise solution [here](https://github.com/ros-industrial/industrial_training/blob/2f9a48969ee8da2e4f4d18cca1a408f2bc6b32ba/exercises/3.2/src/myworkcell_core/src/vision_node.cpp#L8-L10)

```suggestion
    #include <tf2_ros/buffer.h>
    #include <tf2_ros/transform_listener.h>
    #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
```